SimCalorimetry/EcalEBTrigPrimProducers/plugins/EcalEBTrigPrimPhase2Producer.h

```diff
@@ -0,0 +1,96 @@
+#ifndef EcalEBTrigPrimPhase2Producer_h
```

Comment:
This class definition should rather be in the .cc file.

Reply:
Same question as before. Why ?

Reply:
```suggestion
#ifndef SimCalorimetry_EcalEBTrigPrimProducers_plugins_EcalEBTrigPrimPhase2Producer_h
```
If this must be in a separate file even if it is a plugin please add the full path in the include guard.

But consider also this comment https://github.com/cms-sw/cmssw/pull/42633/files#r1355328981